content/en/synthetics/guide/_index.md

```diff
@@ -5,7 +5,14 @@ private: true
 ---
 
 {{< whatsnext desc="General Guides:" >}}
-    {{< nextlink href="synthetics/guide/uptime-percentage-widget" >}}Website Uptime monitoring with SLO{{< /nextlink >}}
+    {{< nextlink href="synthetics/guide/identify_synthetics_bots/" >}}Identify Synthetics bots{{< /nextlink >}}
+{{< /whatsnext >}}
+
+{{< whatsnext desc="API Tests:" >}}
+    {{< nextlink href="synthetics/guide/uptime-percentage-widget" >}}Website uptime monitoring with SLOs{{< /nextlink >}}
```

Comment:
Love the split! The `Website uptime monitoring with SLOs` guide showcases an API test but is actually true for both types of tests so I'd rather have it in the general guides section